docs/release-notes/15-0-0/README.md

```diff
@@ -98,12 +100,14 @@ Before version 15.0, the design configuration in the administration was very com
 - Main menu font on hover
 - Main menu border
 
-All these colors will now be calculated depending on the brightness of the respective background to ensure a high-enough contrast. Font colors will either be black or white, and border colors will be set only if there is a bright background.
+All these colors will now be calculated depending on the brightness of the respective background to ensure a high-enough contrast. Font colors will either be black or white, and border colors will be set only if there is a bright background. [Read more about design options for Enterprise customers in our system admin guide](../../system-admin-guide/design/).
 
 ### Reduce manual cleanup when adding a custom field to a type – no more auto-applying to all projects
 
 With OpenProject 14.6, we released a feature that allows you to enable or disable a custom field for multiple projects at once. This is why with OpenProject 15.0, we remove the automation to apply a new custom field to all projects where the respective type is activated. This reduces manual cleanup in case you did not want to activate the new custom field in all projects. If you do want that, you can use the feature we introduced last release and go to *Administration > Custom Fields* and click on the 'Add projects' button.
 
+[Read more about custom fields in our system admin guide](../../system-admin-guide/custom-fields/).
+
 ### Improved navigation clarity – 'My account' is renamed to 'Account settings'
 
 In the personal menu that can be accessed by clicking on your avatar, we renamed 'My account' to 'Account settings', in order to give you a clearer understanding that this menu item contains settings. It now also differs more clearly from 'My Page' and 'My Activities‘, which provide personal data instead of settings.
```

Comment:
```suggestion
In the personal menu that can be accessed by clicking on your avatar, we renamed 'My account' to '[Account settings](../../user-guide/account-settings/)', in order to give you a clearer understanding that this menu item contains settings. It now also differs more clearly from 'My Page' and 'My Activities‘, which provide personal data instead of settings.
```